src/pages-and-resources/pages/PageGrid.jsx

```diff
@@ -20,19 +18,9 @@ function PageGrid({ intl, pages }) {
   );
 }
 
-const pageShape = PropTypes.shape({
-  id: PropTypes.string.isRequired,
-  title: PropTypes.string.isRequired,
-  isEnabled: PropTypes.bool.isRequired,
-  showSettings: PropTypes.bool.isRequired,
-  showStatus: PropTypes.bool.isRequired,
-  showEnable: PropTypes.bool.isRequired,
-  description: PropTypes.string.isRequired,
-});
-
```

Comment:
This seems to have duplicate definitions. Two already existed and I created another one!